pandas/io/parsers/readers.py

```diff
@@ -709,14 +743,27 @@ def _read(
     # Check for duplicates in names.
     _validate_names(kwds.get("names", None))
 
-    # Create the parser.
-    parser = TextFileReader(filepath_or_buffer, **kwds)
+    extensions = kwds.get("extensions", None)
```

Comment:
is there a way to do this without adding all these new keywords?

Reply:
Yes, `extensions` and `glob` are extra features, but not necessary.